### DISCUSSION TITLE
"Natural language" due dates

### DISCUSSION THREAD

**Discussion by swantzter** [ransome1/sleek#531] [Ideas]
It would be really nice if there was some way to input due dates in the free text field without having to write it as a YYYY-MM-DD date, but that would then be converted to a YYYY-MM-DD date on save. Similar to [Todoist](https://todoist.com/).

I'm not sure what would make most sense, especially if part of this input would be keeping true to the todo.txt format of no spaces in tags.

being able to say things like "1 mar" for next match 1. or "next wed"/"next wednesday" for next wednesday would be golden, but maybe to avoid spaced you could make some more "cryptic" way of writing it, as long as you don't have to look at a calendar I don't really care about the format. One example could be "due:tomorrow", "due:+2d" (for overmorrow), "due:wed" (for wednesday) "due:wed+1" (for next wednesday)

**Comment by ransome1** [comment]
@swantzter I'm not a fan of a cryptic or sleek proprietary language, but cases like `next monday` or `tomorrow` or `end of the year`, are already possible: https://github.com/ransome1/sleek/wiki/Available-todo.txt-attributes-and-extensions#friendlyspeaking-date-names

Feel free to just try it out. Of course you need to prepend `t:` or `due:` to it.

**Reply by swantzter** [reply]
Oh I somehow missed that when searching around. Neat
